src/samples/EducationStart/index.tsx

```diff
@@ -262,6 +272,18 @@ const EducationStartCase: FunctionComponent<any> = () => {
 
   if (!isLoggedIn || shuttered === null) {
     return null;
+  } else if (currentDisplay === 'servicenotavailable') {
+    return (
+      <>
+        <AppHeader appname={t('EDUCATION_START')} hasLanguageToggle={false} />
+        <div className='govuk-width-container'>
+          <MainWrapper showPageNotWorkingLink={false}>
```

Comment:
This can be removed, ServiceNotAvailable is already having main container wrapper

Reply:
Its removed